From 4b1c4848ec99e30490916f8d5302b92254a0c133 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Mon, 17 Sep 2018 12:59:26 +0200
Subject: [PATCH] Merge pull request #187 in SW_UGW/linux from UGW_SW-30440-8.x
 to master

* commit '51502a7eb2f5171e2faac23632f8c9aabfe6f376':
  Introduce Flag to Identify Non Ethernet WAN Sessions in PPA
---
 include/uapi/net/ppa_api.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/net/ppa_api.h b/include/uapi/net/ppa_api.h
index 6d2d578dc607..6e72f8bbd71f 100755
--- a/include/uapi/net/ppa_api.h
+++ b/include/uapi/net/ppa_api.h
@@ -491,6 +491,7 @@
 /* Container Session Flags */
 #define SESSION_FLAG2_VETH 0x00001000
 #define SESSION_FLAG2_CONTAINER 0x00002000
+#define SESSION_FLAG2_NON_ETHWAN_SESSION 0x00004000
 /* Other flags */
 #define FLG_PPA_PROCESSED			0x100	/* this used to mark ecah packets which are processed by ppa datapath driver*/
 #define SESSION_FLAG_TC_REMARK		 0x40000000 /*Flag to sepcify bit 30 in extmark which specifies packet classified by iptables when set to 1*/
